views/index.handlebars

```diff
@@ -13,18 +13,24 @@
         <div class="bg-dark-blue text-light">
 
           {{#if (eq dataset "makers")}}
-            <h4>
+            <h4 class="mt-0 i18n" data-i18n="makers-ppe-cta">
               Make PPE &amp; get it into the hands of those who need it.
             </h4>
-            <p>
-              Coordinate with a Maker Group near you. Questions? Read the <a class="text-light" href="/faq#section-makers">Maker FAQ</a>.
+            <p class="i18n" data-i18n="[html]maker-request-near-me">
```

Comment:
let's be consistent about the prefix for `makers` i18n here. @nburt I'm not *that* familiar with the jQuery i18n library -- should these be `ftm-makers-blahblahblah` or is Linda taking the right approach by replacing `ftm` with `makers` as the prefix here?

Reply:
My thinking/prediction was that Maker-only features and thus code demand will outpace the base ftm code given the complex nature of the Makerspace ecosystem IRL. Just attempting to save key strokes :) but happy to whatever

Reply:
Let's keep as is for consistency, can you also update them to all be lowercase too?

All keys should be "ftm-<key in all lowercase characters".